rust/value.rs

```diff
@@ -328,6 +329,7 @@ impl fmt::Display for DurationLiteral {
                     Some(time) => write!(f, "T{time}")?,
                 }
             }
+            DurationLiteral::Time(time) => write!(f, "T{time}")?,
```

Comment:
Should this not be PT?

Reply:
`"P"` is written unconditionally at the start of formatting.